pkgs/development/tools/analysis/vera/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python27Full
+, boost
+, tcl
+}:
+
+stdenv.mkDerivation {
+  pname = "vera";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
```

Consider using the [upstream bitbucket](https://bitbucket.org/verateam/vera/wiki/Home). The Github is just a replication

```suggestion
  src = fetchFromBitbucket {
```